src/commands/create-cluster.yml

```diff
@@ -29,13 +29,13 @@ parameters:
   service-principal:
     description: |
       Service principal used for authentication to Azure APIs.
```

Comment:
I think it would be great if we could also update the description for the parameter to reflect the change e.g. "Name of environment variable that stores the service principal used for authentication to Azure APIs."